src/app/components/search-facet/search-facet.component.html

```diff
@@ -0,0 +1,35 @@
+<onto-search-facet
+  [data]="data"
+  [type]="type"
+  [facetTitleTemplate]="useCustomTemplate ? customTitleTemplate : facetTitleTemplate"
```

Comment:
As long as you don't have a default template builtin the facet and the template is mandatory input, why do you need switch the template reference conditionally? What are you testing with this?

Reply:
removed templates and test